src/Window.js

```diff
@@ -476,6 +477,9 @@ const _makeWindow = (options = {}, parent = null, top = null) => {
       if (nativeVr && nativeVr.VR_IsHmdPresent()) {
         result.push(window[symbols.mrDisplaysSymbol].vrDisplay);
       }
+      if (oculusVr && oculusVr.Oculus_IsHmdPresent()) {
+        result.push(window[symbols.mrDisplaysSymbol].vrDisplay);
```

Comment:
If both are present that would mean we have the same VR display twice? It might make more sense to make the oculus device completely separate from the OpenVR one.

Reply:
I changed this. If an Oculus headset is detected it takes precedence over OpenVR